tests/vm_test.go

```diff
@@ -676,10 +676,11 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					Expect(err).ToNot(HaveOccurred())
 					Expect(newVM.Spec.RunStrategy).ToNot(BeNil())
 					Expect(*newVM.Spec.RunStrategy).To(Equal(v1.RunStrategyHalted))
+					By("Ensuring stateChangeRequests list is cleared")
 					Expect(len(newVM.Status.StateChangeRequests)).To(Equal(0))
 				})
 
-				It("should restart a running VM", func() {
+				It("[test_id:2187] should restart a running VM", func() {
```

Comment:
I am missing the action of restart the VM and
wait for VMI to be restarted. observe the UUID has changed to be sure it's really a new instance verify VM's runStrategy is still RerunOnFailure verify vm.status.stateChangeRequests is not present in vm's manifest

Reply:
Confirmed test is correct in offline conversation with @ILpinto